docs/DeveloperGuide.adoc

```diff
@@ -812,18 +812,20 @@ These instructions only provide a starting point for testers to work on; testers
 .. Copy the jar file to the folder you would like to use as a home address for Delino application
    Expected: Shows the GUI of the Delino App. The window size may not be optimum
 
-=== Adding an order
+=== Inserting an order
 
-. Add a minimum of 2 orders
+. Insert a minimum of 2 orders
 
-.. Add command format: `add` `tid/TRANSACTION_ID` `n/NAME` `a/ADDRESS` `p/PHONE_NUMBER` `ts/DELIVERY_DATE_&_TIME` `w/WAREHOUSE_LOCATION` `pm/PAYMENT_METHOD` `[c/COMMENTS_BY_CUSTOMER]` `[type/TYPE_OF_ITEM]`
+.. Insert command format: `insert` `tid/TRANSACTION_ID` `n/CUSTOMER_NAME` `a/ADDRESS` `p/PHONE_NUMBER` `ts/DELIVERY_DATE_&_TIME` `w/WAREHOUSE_LOCATION` `pm/PAYMENT_METHOD` `[c/COMMENTS_BY_CUSTOMER]` `[type/TYPE_OF_ITEM]`
```

Comment:
pm/PAYMENT_METHOD - I think we have discussed that this should change to CASH_ON_DELIVERY, sorry I did not state it down in the telegram message.
Perhaps the prefix can change to cod. If you do decide to change please also change it in the prefix appendix! and refactor it accordingly, Thank you!